### PR TITLE
check for changes on remote origin first

### PR DESCRIPTION
**Pull request by cjyetman**

### CODE REVIEW

**Comment by cjyetman** [comment]
had to set the directory for the git fetch, but now it should work as expected

File: test_test_cases.R [hunk 1]
```diff
@@ -59,6 +59,7 @@ report_git_status <-
   function(repo_root = ".") {
     for (i in  repo_root) {
       cli::cli_h1(paste0("repo status for: ", i))
+      system2("git", c("-C", i, "fetch"), stdout = FALSE)
```
Comment (maurolepore) [inline test_test_cases.R:62]: Can you explain a bit what this does? I'm scratching my head :)

Comment (maurolepore) [reply test_test_cases.R:62]: Running it in my head give me something that makes little sense:


```bash
git -C <some path> fetch
```

Comment (cjyetman) [reply test_test_cases.R:62]: yes, exactly.... and so it runs the git command as if it were run from `<some path>`

https://git-scm.com/docs/git#Documentation/git.txt--Cltpathgt

Comment (maurolepore) [reply test_test_cases.R:62]: Amazing!

**Review by maurolepore — APPROVED** [review]

**Pull request merged**